### PR TITLE
Upgrade to Python 3

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: zip_loader.py [hunk 1]
```diff
@@ -133,34 +133,32 @@ def detect_changes(data_path, fields, past_hashes, output_hashes, shape_token=No
...
-                hasher.update(float_subber.sub(r'\1', str(row[:attribute_subindex])))  # Hash only attributes
-                if shape_token:
-                    shape_string = row[-1]
-                    if shape_string:  # None object won't hash
-                        shape_string = float_subber.sub(r'\1', shape_string)
-                        hasher.update(shape_string)
-                    else:
-                        hasher.update('No shape')  # Add something to the hash to represent None geometry object
-                # Generate a unique hash if current row has duplicates
+            open(hash_store, 'w') as hash_csv:
```
Comment (steveoh) [inline zip_loader.py:136]: is the indention correct here?

File: zip_loader.py [hunk 2]
```diff
@@ -516,36 +514,36 @@ def upload_zip(source_name, output_directory):
...
 
     spec_manager.save_spec_json(feature)
 
 
 def delete_feature(source_name):
     """Delete a feature, remove it from packages and delete all it's files on drive."""
-    print 'Deleting', source_name
-    confirm_delete = raw_input('Are you sure you want to permanently delete files (yes, no): ')
+    print('Deleting', source_name)
+    confirm_delete = input('Are you sure you want to permanently delete files (yes, no): ')
```
Comment (steveoh) [inline zip_loader.py:529]: this makes it so that python 2 will never work. is that ok?

**Review by steveoh — APPROVED** [review]
looks pretty normal

File: zip_loader.py (hunk 2 above)
Comment (stdavis) [reply zip_loader.py:529]: Yes. I don't see a reason to go back to python 2.

File: zip_loader.py (hunk 1 above)
Comment (stdavis) [reply zip_loader.py:136]: Yes, I believe it's correct because of the line continuation on 135.

**Pull request merged**